src/main/java/rostyslav/ludchenko/ImplementsRunnable.java

```diff
@@ -0,0 +1,18 @@
+package rostyslav.ludchenko;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ImplementsRunnable implements Runnable {
+    private static final Logger LOGGER = LogManager.getLogger(ImplementsRunnable.class);
+    Counter counter = new Counter();
```

Comment:
```suggestion
    Counter counter;
```
Counter потрібно створити в мейні і передати його через конструктор сюди. 
Аналогічно і в іншому потоці

Reply:
Done